### PR TITLE
#1901 move indicator helpers

### LINKED ISSUE #1901: Migrate indicator helpers to datatable utilities
## Is your feature request related to a problem? Please describe.

Indicator helpers are all imported via `utilities/getIndicatorTableData`. For consistency with other helpers and for reduced imports/dependency cycles, these should be moved to `dataTableUtilities`.

## Describe the solution you'd like

See above.

## Implementation

Migrate `getIndicatorTableData` methods to `Indicator` object classes.

## Describe alternatives you've considered

N/A.

## Additional context

N/A.

### PR DESCRIPTION
**Pull request by wlthomson**
Fixes #1901.

## Change summary

Fixes indicator (and related) dependency cycles.

## Testing

Functional changes only, no need to test.

- [ ] App doesn't crash.

### Related areas to think about

N/A.

### CODE REVIEW

File: src/pages/dataTableUtilities/getPageInfoColumns.js [hunk 1]
```diff
@@ -71,25 +71,25 @@ const PAGE_INFO_ROWS = (pageObject, dispatch, route) => ({
   theirRef: {
     title: `${pageInfoStrings.their_ref}:`,
     info: pageObject.theirRef,
-    onPress: () => dispatch(PageActions.openModal(MODAL_KEYS.THEIR_REF_EDIT, route)),
+    onPress: () => dispatch(ACTIONS.openModal(MODAL_KEYS.THEIR_REF_EDIT, route)),
```
Comment (joshxg) [inline src/pages/dataTableUtilities/getPageInfoColumns.js]: Could be wrong, but this is the action constant, not the action creator. If you import `BasePageActions` from `getPageActions`? (This is a mess, I'm sorry :( )

File: src/utilities/index.js [hunk 2]
```diff
@@ -18,15 +18,6 @@ export { backupValidation } from './fileSystem';
 export { debounce } from './underscoreMethods';
 export { getModalTitle, MODAL_KEYS } from './getModalTitle';
 // eslint-disable-next-line import/no-cycle
```
Comment (joshxg) [inline src/utilities/index.js]: this disable still needed?

File: src/pages/dataTableUtilities/getPageInfoColumns.js (hunk 1 above)
Comment (wlthomson) [reply src/pages/dataTableUtilities/getPageInfoColumns.js]: Whoops, 789ac842411e8eb93fb69f38bf6d08f9efc8a0f7.

File: src/utilities/index.js (hunk 2 above)
Comment (wlthomson) [reply src/utilities/index.js]: Unrelated to indicators but still needed as there is a cycle for the `finalisation` import (`UIDatabase`) 😢 .

**Pull request merged**